applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -72,8 +72,8 @@ return view.extend({
 
 		var m, s, o;
 
-		m = new form.Map('upnpd', [_('Universal Plug & Play')],
-			_('UPnP allows clients in the local network to automatically configure the router.'));
+		m = new form.Map('upnpd', [_('UPnP IGD & PCP/NAT-PMP')],
+			_('UPnP IGD & PCP/NAT-PMP allows clients in the local network to automatically configure the router.'));
```

Comment:
it's better to use "and" instead of &

Reply:
Search for ["when to use ampersand vs and"](https://www.google.com/search?q=when+to+use+ampersand+vs+and) says:
> In formal writing like academic papers or business correspondence, spell out the word and instead of using the ampersand. The exception to this rule is if the ampersand is part of an official title, such as a company name.


Translators may be afraid to translate the & and leave it as is.

Reply:
I think in screen and menu real-estate, we should be more efficient. Although I agree with the stylistic finesse. One could technically get away with 'n' but that might take a while to get used to.